alchemy/src/cloudflare/website.ts

```diff
@@ -189,24 +174,23 @@ export default {
           ? {
               command: props.dev.command,
               url: props.dev.url,
-              cwd,
             }
           : undefined,
       } as WorkerProps<any> & { name: string };
 
       if (wrangler) {
-        // paths in wrangler.jsonc must be relative to it
-        const relativeToWrangler = <S extends string | undefined>(f: S): S =>
-          (f ? path.relative(path.dirname(wranglerJsonPath), f) : f) as S;
+        const wranglerPath = path.relative(cwd, wranglerJsonPath);
+        const wranglerDir = path.dirname(wranglerPath);
+
         await WranglerJson("wrangler.jsonc", {
-          path: wranglerJsonPath,
+          path: wranglerPath,
           worker: workerProps,
-          main: relativeToWrangler(wranglerMainPath),
+          main: mainPath ? path.relative(wranglerDir, mainPath) : undefined,
```

Comment:
Why would it be undefined?

Reply:
Ah I see. It's allowed for there to be no worker? Like a Pages only solution

Reply:
This is already implemented (I copied that typedef from the actual file). If there's no `main` or `entrypoint`, the website component already passes a generic `script` property to the 

I can't tell if it actually works though, let me double check.